### DISCUSSION TITLE
How to setup vuetorrent plugin in Docker

### DISCUSSION THREAD

**Discussion by fdfytr** [VueTorrent/VueTorrent#615] [General]
Had to use search and dig [through links from FAQ](https://github.com/WDaan/VueTorrent/issues/27#issuecomment-683424673)

In order to install plugin in docker:
1) download zip from [releases](https://github.com/WDaan/VueTorrent/releases)
2) unzip and upload on your server (i.e. /docker/config)
3) create an additional volume in qbittorrent container
container -> /vuetorrent
host -> /docker/config/vuetorrent
4) put location for webui in settings>webui as /vuetorrent
5) save & restart the container

p.s. if you stuck at 'Unacceptable file type, only regular file is allowed' error during above steps
- stop container
- open qBittorrent/qBittorrent.conf in container settings
- change WebUI\AlternativeUIEnabled=true to WebUI\AlternativeUIEnabled=false
- restart container

**Comment by WDaan** [comment]
Thanks for the write up 😃
